doc/deployment/deploy-to-azure-container-service-using-jenkins.md

```diff
@@ -0,0 +1,98 @@
+# Deploy to Azure Container Service using Jenkins
+
+This document shows how to deploy this todo app java project to Kubernetes cluster using Jenkins.
+
+## Create Azure services
+
+You can create the Azure Services using [Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
+
+### Create Azure Container Service
+
+1. Create a resource group
+
+    ```bash
+    az group create -n <your-resource-group-name> -l westeurope
+    ```
+
+1. Create Kubernetes cluster
+
+    ```bash
+    az acs create --orchestrator-type kubernetes -g <your-resource-group-name> -n <your-kubernetes-cluster-name> --generate-ssh-keys
+    ```
+
+1. Connect to the cluster, this command download the Kubernetes configuration to your profile folder. The kubectl will use this configure file to interact with your Kubernetes cluster.
+
+    ```bash
+    az acs kubernetes get-credentials -g <your-resource-group-name> -n <your-kubernetes-cluster-name>
+    ```
+
+1. Install the kubectl command line
+
+    ```bash
+    az acs kubernetes install-cli
+    ```
+
+### Create Azure Container Registry
+
+1. Run below command to create an Azure Container Registry.
+After creation, use `login server` as Docker registry URL in the next section.
+
+   ```bash
+   az acr create -n <your-registry-name> -g <your-resource-group-name>
+   ```
+
+1. Run below command to show your Azure Container Registry credentials.
+You will use Docker registry username and password in the next section.
+
+    ```bash
+    az acr credential show -n <your-registry-name>
+    ```
+
+## Configuration
+
+1. Install "EnvInject" plugin.
```

Comment:
Also mention other prerequisites, including maven, docker and kubernetes-cd plugin